infrahub_sdk/spec/menu.py

```diff
@@ -28,7 +28,7 @@ def spec(self) -> InfrahubMenuFileData:
         return self._spec
 
     def validate_content(self) -> None:
-        super().validate_content()
+        InfrahubFile.validate_content(self)
```

Comment:
Why are we making this change? Swapping from inheriting from `InfrahubFile` and moving to `ObjectFile` instead but we override the spec method from ObjectFile completely and then call the .validate_content on InfrahubFile I assume to avoid using the one from `ObjectFile`. Am I missing something here?

Reply:
I agree this isn't ideal, and it calls for a larger redesign / cleaned of the `ObjectFile` / `YamlFile` / `MenuFile` but I think this is a larger effort and I would prefer to manage that separately
I will open a dedicated issue to track that

Reply:
Sounds good to me